src/lib/extract/types.ts

```diff
@@ -9,6 +9,7 @@ import type React from 'react';
 
 export type NumberProp = string | number;
 export type NumberArray = NumberProp[] | NumberProp;
+export type BooleanProp = boolean | 'true' | 'false';
```

Comment:
Why can it be string too?

Reply:
Exactly same reason as NumberProp can be string. 
Example component should be valid:
```tsx
  <Filter id="convolveMatrix2" x="0" y="0" width="100%" height="100%">
    <FeConvolveMatrix
      kernelMatrix="1 -1 2 0 0 0 0 0 -2"
      preserveAlpha="false" />
  </Filter>
```